src/js/package.json

```diff
@@ -31,6 +31,7 @@
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
     "prettier": "^2.2.1",
+    "sinon": "^18.0.0",
```

Comment:
What's sinon?

Reply:
> What's sinon?

@hoodmane It's a test helper library for mocking / spying function call. 

Personally, I would like to switch our JS testing library from mocha to Jest or Vitest. Mocha requires `chai` for assertion and `sinon` for mocking, while Jest and Vitest have those features themselves, so we don't need to care about them.

Reply:
If you want to switch to any JS testing library you prefer, feel free.